### ISSUE TITLE
Heroku server hosting the online demo is no longer operational.

### ISSUE THREAD

**Issue by GemedetAdept** [tomtung/latex2unicode#23]
When visiting the online demo webpage (https://latex2unicode.herokuapp.com/), it returns an “Application error”, saying:

> An error occurred in the application and your page could not be served. If you are the application owner, check your logs for details.

I'm uncertain if this project is still being worked on. If it's not, I apologize for effectively necroing.

**Comment by tomtung** [comment]
Yeah Heroku stopped their free tier and the demo app broke. I haven't got around to fix it, but the library should be usable.
